ContractsRevo/agricultural-quality-contract/README.md

```
@@ -2,6 +2,9 @@
 
 A stellar smart contract for managing agricultural product quality control, verification, and dispute resolution on the Stellar network.
 
+- contractId: CDZ7KPCB4XP45NGVKWIEAENEJIEX63EOPF6TJ77VWQXN53BGP5TRBC3F
+- Link: https://stellar.expert/explorer/testnet/contract/CDZ7KPCB4XP45NGVKWIEAENEJIEX63EOPF6TJ77VWQXN53BGP5TRBC3F
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)